web/docs/operators/transformations/unflatten.md

```diff
@@ -0,0 +1,74 @@
+# unflatten
+
+Unflattens data structures whose field names imply a nested structure.
+
+## Synopsis
+
+```
+unflatten [<separator>]
+```
+
+## Description
+
+The `unflatten` operator creates nested records out of record entries whose
+names include a separator, thus unflattening
+
+:::info
+`unflatten` uses a heuristic to determine the unflattened schema. Thus, the
+schema of a record that has been flattened using the [`flatten`](flatten.md) operator and
+unflattened afterwards may not be identical to the schema of the unmodified
+record.
+
```

Comment:
```suggestion
```

Reply:
Fixed in https://github.com/tenzir/tenzir/pull/3304/commits/50a4ca3a39b0c885e41cd85e91723a405ae5de82